Sources/TuistGenerator/Settings/DefaultSettingsProvider.swift

```diff
@@ -38,6 +38,7 @@ public final class DefaultSettingsProvider: DefaultSettingsProviding {
         "GCC_NO_COMMON_BLOCKS",
         "PRODUCT_NAME",
         "VALIDATE_PRODUCT",
+        "CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER",
```

Comment:
I think this can be kept as a recommended setting rather than an essential setting _(i.e. omitted from this list)_ - from what I gather about this setting projects can still build fine without addressing the associated warnings, though recommended to follow it.

Reply:
The fixture is failing if I don't include it:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/5576161/116415393-ac2a7d00-a839-11eb-862f-4d34fb61c083.png">
Should I leave it in? 🤔

Reply:
We'd need to also omit it from the [expected essential settings](https://github.com/tuist/tuist/pull/2828/files#diff-3c37b394cf43bf806d56d60dcad61e87b619decdc847277eec16934c9adf9156R55) and update the counts accordingly 🤐

Reply:
The way default settings work, we treat all the settings provided by `XcodeProj` as "recommended", however treat an explicit subset of them as "essential", this is controlled via https://docs.tuist.io/manifests/project#defaultsettings

Reply:
I see, so only the count has to change, as it's implicitly there as "recommended".